packages/zoe/src/state.js

```diff
@@ -60,14 +60,23 @@ const makeOfferTable = () => {
         });
       },
       isOfferActive: offerHandle => table.has(offerHandle),
-      deleteOffers: offerHandles =>
-        offerHandles.map(offerHandle => table.delete(offerHandle)),
+      getNotifier: offerHandle => table.get(offerHandle).notifier,
```

Comment:
Is there a reason to expose this? If not, we shouldn't.

Reply:
I was going to explain why clients should have access to notification, but the right answer is to provide access to the notifications, and not the notifier itself. Thanks for noticing that.

Reply:
Okay, I see what you mean. Zoe uses the table row directly, so it doesn't need a method to get just the notifier.

Why is it better that Zoe does the extraction starting from the entire stored row than having the table add a custom method that returns just the notifier (or update facet)?

Reply:
That was a choice by @erights - we used to have a lot of methods like this, but Mark thought it was better to return a whole record.

Reply:
Let's revisit that choice by @erights ;)